README.md

```diff
@@ -1,14 +1,246 @@
-# Wallet Adapter
+# <center>Solana Wallet Adapter </center>
```

Comment:
As a general point, please just stick to markdown and don't rewrite this all in HTML. It does render OK on Github but markdown is simpler and more consistent.